ci/e2e/terraform_provider_ocm_fullcycle_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	client "github.com/openshift-online/ocm-sdk-go"
 	"github.com/openshift-online/ocm-sdk-go/logging"
 	"github.com/segmentio/ksuid"
-	"github.com/terraform-redhat/terraform-provider-ocm/ci/helper"
+	"github.com/terraform-redhat/terraform-provider-rhcs/ci/helper"
```

Comment:
Please also change the const name (line 28) terraformProviderOCMFilesDir -> terraformProviderRhcsFilesDir
"terraform_provider_ocm_files" -> "terraform_provider_rhcs_files"